src/Info.test.js

```diff
@@ -0,0 +1,19 @@
+import { screen, waitFor } from '@testing-library/react';
+import React from 'react';
+import Info from './Info';
+import { render } from './setupTests';
+
+describe('Info tests', () => {
+  it('should appear error', async () => {
+    render(<Info error />);
+    expect(await screen.findByTestId('info-error')).toBeInTheDocument();
```

Comment:
`await screen.findByTestId('info-error')` would work as well, no need for expect, if the selector finds nothing it fails. Same below